templates/DatasetMonitoring/projects.html.twig

```diff
@@ -4,9 +4,13 @@
         <span class="header-inst-name">({{instName}})</span>
     {% endif %}
     {% if not pdf | default %}
-        <a href="/html2pdf?url={{ url(app.request.get('_route'), {'id': id, 'renderer': 'html2pdf'}) }}&format=Letter&orientation=landscape&margin=1cm&download=true&filename={{pdfFilename}}" class="download-pdf" title="save as pdf">
-            <img src="{{ asset('build/images/pdf_icon.png') }}">
-        </a>
+        <span class="awesome-icon awesome-icon-pdf" data-url="/html2pdf?url={{ url(app.request.get('_route'), {'id': id, 'renderer': 'html2pdf'}) }}&format=Letter&orientation=landscape&margin=1cm&download=true&filename={{pdfFilename}}">
```

Comment:
Why aren't these just Anchors?

Reply:
One is a POST request, and I wanted both icons to be consistent. Plus for anchor tags, had to remove the default styling that was getting applied to the font awesome icons

Reply:
It could be a GET request as well I guess? It took me a second the figure out what the javascript was doing, I feel it's better if we could do without. If that's possible?

Reply:
Actually, it's probably fine the way it is.